.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,14 @@
+{
+  "image": "grafana/loki-build-image:0.27.1",
+  "containerEnv": {
+    "BUILD_IN_CONTAINER": "false"
+  },
+  "customizations": {
+		"vscode": {
+			"settings": {},
+			"extensions": ["golang.go"],
```

Comment:
Should we add the [Grafana Jsonnet](https://github.com/grafana/vscode-jsonnet) extension?

Reply:
If it works, why not?